### DISCUSSION TITLE
Having no luck with custom CSS

### DISCUSSION THREAD

**Discussion by KenK73** [Lissy93/dashy#1046] [Help and Support]
I am running Dashy v2.1.1 in a Docker container.  I am able to tweak the 'Material' theme to my needs, but I want to make these settings default for everyone.  I saved the modified theme CSS settings and made a custom entry in the  'user-defined-themes.scss' file, and added it to the conf.yml in both the the appConfig.cssThemes and appConfig.theme settings.  The theme shows up in the list, but it will not load the settings. It keeps loading the settings from the default theme. If I select the custom theme, the html element gets modified with the primary/background/background-darker settings of the default theme and thats it.  It wont even let me force the colors for the custom theme in the conf.yml.  I ran my custom CSS through the W3C verifier and its 100% valid.  I have gone over the theme instructions in the wiki carefully and I'm still missing it.

I could really use some help.

**Comment by KenK73** [comment]
It gets worse... So I decided for now I can just edit the default Material theme with my settings and use that until I figure out whats going on with the custom CSS. I went into the container and edited the /src/styles/color-themes.scss file, replacing the color definitions with my own, and then closed the file, exited the container and restarted it.  Nothing changed!  It's still using its default primary color and I have no idea where its getting that value. I have purged my history, cleared cache, used new browsers... it just acts like the file was never changed. What is this witchcraft?

**Reply by silkyplover** [reply]
Same for me. I can confirm this strange behaviour. No clue, what is going on.

**Reply by BobbieERay** [reply]
Same here. I've tried everything, but it feels like the *.scss files are just being ignored. Whatever you do to them, the default themes, with their default values always show up.

**Comment by BobbieERay** [comment]
There appears to be a bug related to this. However, it has been open for over a year now. https://github.com/Lissy93/dashy/issues/1242

**Comment by KenK73** [comment]
I switched to Homer and that has it benefits and drawbacks. It is sufficient for my purpose.

**Reply by BobbieERay** [reply]
I eventually switched to Homepage. Which appears to be in active development still and is sufficient for me also

**Comment by lokmjuhb** [comment]
I encountered the same problem with both local custom css or external style sheet. I wonder if the bug is solved yet?
